optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -174,3 +189,27 @@ The following example finds a computer which runs the most power-demanding proce
     }
 ----
 
+
+[[constraintStreamsImplementations]]
+== Variant implementation types
+
+Constraint streams come in two flavours, a default implementation using Drools as a backend and a pure Java-based
+implementation called "Bavet".
+Drools-based implementation is more feature-complete and it is our primary focus. It will be seeing further improvements
+in future releases.
```

Comment:
Can we remove "and it is our primary focus. It will be seeing further improvements
in future releases."? We should avoid statements about the future work in the documentation whenever it is reasonable. Such statements:
1) raise expectations
2) Don't tend to age well: they implicitly need to be removed or adjusted - even if works goes according to plan.